WalletWasabi.Tests/UnitTests/UserInterfaceTest/PocketSelectionTests.cs

```diff
@@ -103,7 +104,7 @@ public ICoinsView Unspent()
 		throw new NotImplementedException();
 	}
 
-	public bool TryGetByOutPoint(OutPoint outpoint, out SmartCoin? coin)
+	public bool TryGetByOutPoint(OutPoint outpoint, [NotNullWhen(true)] out SmartCoin? coin)
```

Comment:
Make sense, but why in this PR?

Reply:
Switching between branches takes time too. It's faster and I don't think it hurts too much. I can also remove it but ultimately I think that fixing some warnings along the way is not a bad thing.

Reply:
I was just curious, don't need to remove it.